<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Persons:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tracks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="databaseHistogramCount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="databaseTrackCount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="databasePersonCount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="databaseSize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms / track:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="databaseAvgHistogramsPerTrack">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms / peson:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="databaseAvgHistogramsPerPerson">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Last recognition result</string>
    </property>
    <widget class="QLabel" name="personImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tracks:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="personId">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="personTrackCount">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="personHistogramCount">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="personSize">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="personNew">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>NEW</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Video processing</string>
    </property>
    <widget class="QPushButton" name="processingButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sourceFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="sourceCameraButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="sourceFileButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectFileButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select file...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string extracomment="Learns new faces and recognizes already learned faces on the fly while processing videos.">Recognize and learn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="Recognizes already learned faces on the fly while processing videos.">Recognize only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string extracomment="Detected face tracks are not added to the database. Recognition result is shown after all frames of a track are processed (track is lost).">Test mode</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDatabase">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Database operations</string>
    </property>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="emptyButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mergeButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Merge</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Merge persons - all tracks from the persons are combined under the &quot;kept person&quot;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPersonId1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxPersonId2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Person to be merged:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kept person:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Search statistics</string>
    </property>
    <widget class="QLabel" name="searchTime">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="searchHistogramsCompared">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms compared:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Histograms matched:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="searchHistogramsSearched">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
